From ebdc81f3363179e76b610e3f7a8e13fa44191b20 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 1 Aug 2023 13:46:15 -0700
Subject: [PATCH] pep517 workaround

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 python/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index e656310..f1a3a42 100755
--- a/python/setup.py
+++ b/python/setup.py
@@ -306,10 +306,10 @@ def HasLibraryDirsOpt():
   return any(_GetFlagValues('library-dirs=', 'L'))
 
 
-if __name__ == '__main__':
+if True:
   ext_module_list = []
   warnings_as_errors = '--warnings_as_errors'
-  if GetOptionFromArgv('--cpp_implementation'):
+  if True:
     # Link libprotobuf.a and libprotobuf-lite.a statically with the
     # extension. Note that those libraries have to be compiled with
     # -fPIC for this to work.
-- 
2.41.0

