From c01f52ffe11a1190dbaba2d9f0132a83a11f4fc9 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 1 Aug 2023 14:26:59 -0700
Subject: [PATCH] Skip unittest proto generation

---
 python/setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/python/setup.py b/python/setup.py
index f1a3a42..7566b60 100755
--- a/python/setup.py
+++ b/python/setup.py
@@ -199,7 +199,6 @@ class BuildPyCmd(_build_py):
     GenProto('../src/google/protobuf/timestamp.proto')
     GenProto('../src/google/protobuf/type.proto')
     GenProto('../src/google/protobuf/wrappers.proto')
-    GenerateUnittestProtos()
 
     # _build_py is an old-style class, so super() doesn't work.
     _build_py.run(self)
-- 
2.41.0

